llama-index-integrations/llms/llama-index-llms-openai/llama_index/llms/openai/base.py

```diff
@@ -964,18 +963,3 @@ def stream_structured_predict(
         # by default structured prediction uses function calling to extract structured outputs
         # here we force tool_choice to be required
         return super().stream_structured_predict(*args, llm_kwargs=llm_kwargs, **kwargs)
-
-    @dispatcher.span
-    def stream_structured_predict(
-        self, *args: Any, llm_kwargs: Optional[Dict[str, Any]] = None, **kwargs: Any
```

Comment:
I wonder if this was meant to be astream_structured_predict

Reply:
I couldn't find any usage of this, should I bring it back in async fashion?

Reply:
There should be a definition in the base `LLM` class that we can copy yea, its basically just here to force the tool choice with openai (streaming structured outputs are definitely under-utilized, but very cool feature)